src/steps/filter.py

```diff
@@ -155,10 +178,15 @@ def add_new_relay_subscription(self, pubsub_topics, node=None):
         self.node1.set_relay_subscriptions(pubsub_topics)
 
     @allure.step
-    def get_filter_messages(self, content_topic, node=None):
+    def get_filter_messages(self, content_topic, pubsub_topics=None, node=None):
         if node is None:
             node = self.node2
-        return node.get_filter_messages(content_topic)
+        if node.is_gowaku():
+            return node.get_filter_messages(content_topic, pubsub_topics)
+        elif node.is_nwaku():
+            return node.get_filter_messages(content_topic)
+        else:
+            raise NotImplemented("Not implemented for this node type")
```

Comment:
Same as above, and there's more cases like this.

Reply:
Thanks, fixed all of them